README.md

```diff
@@ -0,0 +1,159 @@
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  # Popcorn Flix
+  ![Popcorn Flix](image.png)
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 Popcorn Flix](#-popcorn-flix-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Popcorn Flix<a name="about-project"></a>
+
+The Popcorn Flix App is the Capstone project for React module three where we are required to build an SPA utilizing the concepts we learnt. I used Axios to make calls to a movies API to fetch data, I used Redux( useSelector and useDispatch) to update the state by filtering movies by  title. I can navigate from the home page to a single movie page dynamically using React router, and fetch data with useEffect on page load and with action dispatch, finally, I used Jest and React Testing Library to make component tests and mock API calls.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+![React](https://img.shields.io/badge/React-20232A?style=for-the-badge&logo=react&logoColor=61DAFB)
+![React Router](https://img.shields.io/badge/-React%20Router-316192?style=for-the-badge&logo=react-router)
+![Redux](https://img.shields.io/badge/Redux-593D88?style=for-the-badge&logo=redux&logoColor=white)
+![Tailwind](https://img.shields.io/badge/Tailwind_CSS-38B2AC?style=for-the-badge&logo=tailwind-css&logoColor=white)
+![Jest](https://img.shields.io/badge/-jest-%23C21325?style=for-the-badge&logo=jest&logoColor=white)
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Routing using react-router.
+- App-wide State Management with Redux/Toolkit.
+- Styling with Tailwind CSS.
+- Data-fetching with Axios.
+- Testing with Jest and React Testing Library.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+### Live Demo <a name = "live-demo"></a>
+[Popcorn Flix](https://popcorn-flix.netlify.app/)
+
+### Video Presentation <a name = "video-Presentation"></a>
+[Presentation](https://www.loom.com/share/7e820953b26248f68897f22fca7c3f09?sid=3de3d2bb-47e9-4ad0-95b7-83de7e748dfc)
```

Comment:
- [ ] [OPTIONAL] Great job deploying the project and also resolving the issues withh the video presentation, however, this sectionsa are not present in the Table of caontetnt of the README. Kindly fix this issue in order to have a professionally docummendted README.

Reply:
Thank you, i have resolved it.